# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/fullscreen/api/fullscreen-reordering.html.ini
# Commit: bf27ab557da6
# Full Hash: bf27ab557da6f356bcaed60a596f77ac336b22c6
# Author: Ziran Sun <zsun@igalia.com>
# Date: 2023-05-23 21:47:35
# Regressor Bug: 1832837
# File Overlap Count: 1
# Description:
#   Bug 1832837 - Remove the fullscreen element ready check on fullscreen element descendant. r=edgar
#   
#   https://fullscreen.spec.whatwg.org/#fullscreen-element-ready-check
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D177913
# ==============================================================================

diff -r e117237fcb62 -r bf27ab557da6 testing/web-platform/meta/fullscreen/api/fullscreen-reordering.html.ini
--- a/testing/web-platform/meta/fullscreen/api/fullscreen-reordering.html.ini	Tue May 23 08:49:21 2023 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,4 +0,0 @@
-[fullscreen-reordering.html]
-  expected: ERROR
-  [Requesting fullscreen on A, then B, then A]
-    expected: FAIL